### PR TITLE
Updated README accordingly.

### PR DESCRIPTION
**Pull request by k82cn**
Signed-off-by: Da K. Ma <klaus1982.cn@gmail.com>

**Release note**:
```release-note
None
```

### CODE REVIEW

**Comment by TommyLike** [comment]
LGTM.

File: README.md [hunk 1]
```diff
@@ -1,31 +1,147 @@
-# kube-batch
+# Volcano
 
 [![Build Status](https://travis-ci.org/kubernetes-sigs/kube-batch.svg?branch=master)](https://travis-ci.org/kubernetes-sigs/kube-batch)
 [![Go Report Card](https://goreportcard.com/badge/github.com/kubernetes-sigs/kube-batch)](https://goreportcard.com/report/github.com/kubernetes-sigs/kube-batch)
 [![RepoSize](https://img.shields.io/github/repo-size/kubernetes-sigs/kube-batch.svg)](https://github.com/kubernetes-sigs/kube-batch)
 [![Release](https://img.shields.io/github/release/kubernetes-sigs/kube-batch.svg)](https://github.com/kubernetes-sigs/kube-batch/releases)
 [![LICENSE](https://img.shields.io/github/license/kubernetes-sigs/kube-batch.svg)](https://github.com/kubernetes-sigs/kube-batch/blob/master/LICENSE)
 
+Volcano is system for runnning high performance workloads on
```
Comment (animeshsingh) [inline README.md:9]: It should specify right at the outset its a  batch scheduler, or a scheduler for long running jobs. 'sysem' is very overloaded word

File: README.md [hunk 2]
```diff
@@ -1,31 +1,147 @@
...
+performance workload including:
 
-`kube-batch` is a batch scheduler for Kubernetes, providing mechanisms for applications which would like to run batch jobs leveraging Kubernetes. 
+1. machine learning/deep learning,
+2. bioinformatics/genomics, and 
+3. other "big data" applications.
 
-`kube-batch` builds upon a decade and a half of experience on running batch workloads at scale using several systems, combined with best-of-breed ideas and practices from the open source community.
+These types of applications typically run on generalized domain
+frameworks like Tensorflow, Spark, PyTorch, MPI, etc, which Volcano integrates with.
```
Comment (animeshsingh) [inline README.md:19]: Where are these integrations listed? Where  do we find them?

Comment (k82cn) [reply README.md:19]: xref https://github.com/kubernetes-sigs/volcano/tree/master/example/volcano/integrations/mpi

File: README.md (hunk 1 above)
Comment (k82cn) [reply README.md:9]: volcano is not only a scheduler, it's a batch "system" that include others features.

**Pull request merged**